custom/icds_reports/ucr/reports/custom_sql_mpr_2a_person_cases.json

```diff
@@ -15,347 +15,241 @@
     "title": "MPR - 2a - Person cases (Custom SQL) (Static)",
     "description": "",
     "visible": false,
-    "custom_query_provider": "custom.icds_reports.reports.ucr.MPR2APersonCases",
```

Comment:
![image](https://user-images.githubusercontent.com/1471773/46504490-0f77de00-c7fc-11e8-9bc5-89696cb34065.png)
I do not remember, but it should be removed since that class doesn't exist anymore